BOARD=base no longer builds

```
+ make lm32-firmware
cd build/misoc && python3 make.py -X ../.. -t atlys_base -Ot firmware_filename ../../firmware/lm32/firmware.bin -Op programmer impact build-headers
                __  ___  _   ____     _____
               /  |/  / (_) / __/__  / ___/
              / /|_/ / / / _\ \/ _ \/ /__
             /_/  /_/ /_/ /___/\___/\___/
a high performance and small footprint SoC based on Migen
====== Building for: ======
Platform:  atlys
Target:    atlys_base
Subtarget: MiniSoC
CPU type:  lm32
===========================
make -C firmware/lm32 all
make[1]: Entering directory `/home/travis/build/timvideos/HDMI2USB-misoc-firmware/firmware/lm32'
 CC       isr.o
isr.c: In function ‘isr’:
isr.c:17:18: error: ‘HDMI_IN0_INTERRUPT’ undeclared (first use in this function)
  if(irqs & (1 << HDMI_IN0_INTERRUPT))
                  ^
isr.c:17:18: note: each undeclared identifier is reported only once for each function it appears in
isr.c:19:18: error: ‘HDMI_IN1_INTERRUPT’ undeclared (first use in this function)
  if(irqs & (1 << HDMI_IN1_INTERRUPT))
                  ^
make[1]: *** [isr.o] Error 1
make[1]: Leaving directory `/home/travis/build/timvideos/HDMI2USB-misoc-firmware/firmware/lm32'
make: *** [lm32-firmware] Error 2
The command "/trusty/run.py $PWD/.travis-run.sh" exited with 2.
```

--- a/.travis-run.sh
+++ b/.travis-run.sh
@@ -5,7 +5,7 @@
 set -x
 
 BOARDS="atlys opsis"
-TARGETS="hdmi2usb"
+TARGETS="base hdmi2usb"
 
 for BOARD in $BOARDS; do
 	for TARGET in $TARGETS; do

--- a/.travis-run.sh
+++ b/.travis-run.sh
@@ -2,24 +2,27 @@
 
 . scripts/setup-env.sh
 
-set -x
+set +x
+set -e
 
 BOARDS="atlys opsis"
 TARGETS="base hdmi2usb"
 
 for BOARD in $BOARDS; do
 	for TARGET in $TARGETS; do
-		echo "---------------------------------------------"
+		echo "============================================="
 		echo "- $BOARD $TARGET"
 		echo "---------------------------------------------"
-
 		BOARD=$BOARD TARGET=$TARGET make help
+		echo "---------------------------------------------"
 
 		# FIXME: Add ability to compile gateware.
 
 		BOARD=$BOARD TARGET=$TARGET make lm32-firmware
+		echo "---------------------------------------------"
 		BOARD=$BOARD TARGET=$TARGET make fx2-firmware
-
+		echo "---------------------------------------------"
 		BOARD=$BOARD TARGET=$TARGET make clean
+		echo "============================================="
 	done
 done

--- a/Makefile
+++ b/Makefile
@@ -36,13 +36,7 @@ help:
 	@echo "   PROG=programmer      (current: $(PROG))"
 	@echo " SERIAL=serial port     (current: $(SERIAL))"
 
-ifeq ($(TARGET), base)
-  GATEWARE_DEP =
-else
-  GATEWARE_DEP = lm32-firmware
-endif
-
-gateware: $(GATEWARE_DEP)
+gateware: lm32-firmware
 	cd $(MSCDIR) && $(CMD) --csr_csv $(HDMI2USBDIR)/test/csr.csv clean
 	cp hdl/encoder/vhdl/header.hex $(MSCDIR)/build/header.hex
 	cd $(MSCDIR) && $(CMD) --csr_csv $(HDMI2USBDIR)/test/csr.csv build-csr-csv build-bitstream
@@ -53,9 +47,16 @@ load-gateware:
 release-gateware:
 	cd $(MSCDIR)/build && tar -cvzf ../$(HDMI2USBDIR)/$(BOARD)_$(TARGET)_gateware_$(DATE).tar.gz *.bin *.bit
 
+
+ifeq ($(TARGET), base)
+lm32-firmware:
+	cd $(MSCDIR) && $(CMD) build-headers
+	echo "No lm32-firmware for base target."
+else
 lm32-firmware:
 	cd $(MSCDIR) && $(CMD) build-headers
 	$(MAKE) -C firmware/lm32 all
+endif
 
 load-lm32-firmware: lm32-firmware
 	$(FLTERM) --port $(SERIAL) --kernel=firmware/lm32/firmware.bin --kernel-adr=0x20000000 --speed 115200

